pykeepass/kdbx_parsing/common.py

```diff
@@ -166,6 +166,8 @@ class XML(Adapter):
 
     def _decode(self, data, con, path):
         parser = etree.XMLParser(remove_blank_text=True)
+        # Remove extra content at the end
+        data = data[:data.rfind(b'>') + 1]
```

Comment:
This might work, but I think the correct fix is to unpad inside the `DecryptedPayload` class.

``` python
class DecryptedPayload(Adapter):
    """Encrypted Bytes <---> Decrypted Bytes"""

    def _decode(self, payload_data, con, path):
        cipher = self.get_cipher(
            con.master_key,
            con._.header.value.dynamic_header.encryption_iv.data
        )
        payload_data = cipher.decrypt(payload_data)
        payload_data = CryptoPadding.unpad(payload_data, 16)

        return payload_data
...
```